data/data-pipeline/data_pipeline/etl/score/tests/test_etl_utils.py

```diff
@@ -70,3 +73,265 @@ def test_floor_series():
         match="Argument series must be of type pandas series, not of type list.",
     ):
         floor_series(invalid_type, number_of_decimals=3)
+
+
+def test_compare_to_list_of_expected_state_fips_codes():
+    fips_codes_test_1 = [
```

Comment:
It feels like these would be easier to read if you just imported all the FIPS codes from constants then dropped a few from each list, so it's obvious what's missing.

Reply:
Good point. I'll keep the first one explicit (so the test also checks whether the constants change), but I'll make the others subtractions to improve readability.